packages/lib/packages/delivery/src/Messages.ts

```diff
@@ -143,6 +143,11 @@ export async function incomingMessage(
 
     const envelopWithPostmark: EncryptionEnvelop = {
         ...envelop,
+        metadata: {
```

Comment:
Currently, we're writing encrypted DeliveryInformation in the db. In order to make this feature work we've to write the encrypted struct instead. Do you think that has implications regarding security etc. ??

Reply:
no I think that's fine. If the DS gets hacked the Delivery Information are leaked but they are already contained in the unencrypted `conversationId`